catalogue/views.py

```diff
@@ -67,7 +88,7 @@ def get_context_data(self, **kwargs):
 
 
 class SendInquiry(generic.detail.SingleObjectMixin, generic.edit.FormView):
```

Comment:
restrict to http post
https://docs.djangoproject.com/en/dev/ref/class-based-views/base/#django.views.generic.base.View.http_method_names